Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -471,7 +471,7 @@ public AndConstraint<TAssertions> NotMatch(string wildcardPattern, string becaus
             new StringWildcardMatchingStrategy
             {
                 IgnoreCase = options.IgnoreCase,
-                IgnoreNewLineDifferences = options.IgnoreNewlineStyle
+                IgnoreAllNewlines = options.IgnoreAllNewlines
```

Comment:
`options.IgnoreAllNewlines` mistake?

Reply:
I don't get, what you mean, @ITaluone. Can you go a bit into more detail?

Reply:
Yes :)

You changed `options.IgnoreNewlineStyle` to `options.IgnoreNewlineStyle` but you didn't in the negative assertion (line 599)

Reply:
Ah, you are right.
I completely forgot 😄 

It's now fixed!